src/client/pages/User/hooks/useRolesFields.ts

```diff
@@ -65,14 +73,14 @@ export const useRolesFields = (props: PropsFormDefinition): Returned => {
         shouldShow,
         defaultValue: userRoles.filter((role) => role.role === roleName).map((role) => role.countryIso),
         watches: {
-          clearIf: (props) => {
+          clearIf: (props): { shouldClear: boolean; clearValue: unknown } => {
```

Comment:
can we use return type from types definitions ? e.g. `ReturnType<typeof FieldDefinition['watches']['clearIf']>` ?